README.md

```diff
@@ -94,6 +94,59 @@ The error is normal because the default command is designed for child containers
 FROM openjdk
 ADD my_app_0.0.1.jar app.jar
 ```
+
+# Stackdriver Logging
+The [Google Cloud Stackdrive Logging](https://cloud.google.com/logging/) service may be used
+by applications running in this image either from Google Cloud Platforms or any other platform with network access to the Google Cloud services. 
+
+## Stackdriver Logging Dependencies
+The application deployed on this image must provide the stackdriver logging classes and their 
+dependencies.  This can be done easily in maven or gradle following [these instructions](https://cloud.google.com/logging/docs/reference/libraries#client-libraries-install-java).
+
+## Stackdriver Logging Authentication
+### Local Development
+When running locally, the [Google Cloud SDK](https://cloud.google.com/sdk/) along with [Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials) can be used to authenticate the logging client library with stackdriver logging service.  This ultimately means that after the SDK is installed, the following command is all that is required prior to running the logging client library:
+```shell
+gcloud auth application-default login
+```
+### Google Cloud Platforms
+No additional authentication is required when the stackdriver logging mechanism is deployed on a Google Cloud Platform.
+
+### Other environments
+To use the stackdriver logging client from other platforms see the [Google Cloud Platform Authentication Guide](https://cloud.google.com/docs/authentication#getting_credentials_for_server-centric_flow)
+
+## Stackdriver Configuration
+The Stackdriver API may be [used directly](https://cloud.google.com/logging/docs/reference/libraries#using_the_client_library) by the application, however it is often more convenient to use the [Java Util Logging](https://docs.oracle.com/javase/8/docs/api/java/util/logging/package-summary.html) handler that is provided by the libraries.
+
+To enable Java Util Logging, a `logging.properties` file must be provided and configured in the `java.util.logging.config.file` system property. This can be either added to a custom image or bound onto the standard image using the `-v` option:
+```shell 
+docker run -it --rm \
```

Comment:
@meltsufin I've been trying to come up with some simple programmatic examples for configuring JUL andto put it simply it is just dire!   It is a mess of private static variables initialised from system properties, some of which can be reset, but others that cannot.   When the configuration happens is really crucial and to have full control over that you need to set environment variables anyway.

I think full programmatic configuration of JUL is for super users (perhaps super-human users).

So instead, I'll work on an alternative simple programmatic example that shows how a `logging.properties` file can be loaded from the classpath using programmatic API and system variables.